### PR TITLE
Added convergency test

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1327160. I have also improved the plotting facilities.
The tests are running here: https://ci.openquake.org/job/zdevel_oq-risklib/324

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,5 +1,8 @@
 python-oq-risklib (0.6.0-1) precise; urgency=low
 
+    [Michele Simionato]
```
Comment (ptormene) [inline debian/changelog]: Developer names are indented differently

Comment (micheles) [inline debian/changelog]: You are right, the strange indentation came from the resolution of a merge conflict

File: openquake/commonlib/commands/plot.py [hunk 2]
```diff
@@ -43,31 +44,49 @@ def make_figure(n_sites, imtls, spec_curves, curves=(), label=''):
     return plt
 
 
-def plot(hazard_pik):
+def combined_curves(haz, hazard_pik):
+    """
+    returns: curves_by_rlz, mean_curves
```
Comment (ptormene) [inline openquake/commonlib/commands/plot.py]: You might add the params to the docstring

File: openquake/commonlib/commands/plot.py [hunk 3]
```diff
@@ -43,31 +44,49 @@ def make_figure(n_sites, imtls, spec_curves, curves=(), label=''):
...
+    curves_by_rlz = haz['rlzs_assoc'].combine(haz['curves_by_trt_gsim'])
+    rlzs = sorted(curves_by_rlz)
+    weights = [rlz.weight for rlz in rlzs]
+    return curves_by_rlz, scientific.mean_curve(
+        [curves_by_rlz[rlz] for rlz in rlzs], weights)
+
+
+def plot(hazard_pik, hazard_pik2=None):
     """
     Hazard curves plotter
```
Comment (ptormene) [inline openquake/commonlib/commands/plot.py]: Perhaps also this docstring might be more exhaustive

**Comment by ptormene** [comment]
Apart from the minor comments, LGTM

**Comment by micheles** [comment]
I have updated the docstrings

**Pull request merged**